### PR TITLE
Fix client crashing if loaded files don't exist anymore

### PR DESCRIPTION
**Pull request by theCalcaholic**
This commit wraps the loading of recently opened files into a try-catch statement to prevent crashes on launch.

### CODE REVIEW

File: CSharpMinifier.GUI/frmMain.cs [hunk 1]
```diff
@@ -33,8 +33,14 @@ void FillList(string[] fileNames)
...
 				var lbItem = new ListBoxItem(fileName, Path.GetFileName(fileName));
-				Sources[lbItem.Key] = File.ReadAllText(lbItem.Key);
-				lbInputFiles.Items.Add(lbItem);
+                try
+                {
+                    Sources[lbItem.Key] = File.ReadAllText(lbItem.Key);
+                    lbInputFiles.Items.Add(lbItem);
+                }
+                catch(Exception e)
+                { }
```
Comment (KvanTTT) [inline CSharpMinifier.GUI/frmMain.cs]: Can you use tabs instead of spaces? Project is written in tab-style.

Comment (theCalcaholic) [inline CSharpMinifier.GUI/frmMain.cs]: Ok, sorry for that. :)

Comment (KvanTTT) [inline CSharpMinifier.GUI/frmMain.cs]: @theCalcaholic can you please fix it in this pull request? :) I'll merge it then.

**Comment by KvanTTT** [comment]
Less than a year has passed :)

**Comment by theCalcaholic** [comment]
@KvanTTT Yes, sorry. :D
Totally forgot about it.

**Pull request merged**